### ISSUE TITLE
Braziers fire damage radius too large [$25]

### ISSUE THREAD

**Issue by Stoabrogga** [azerothcore/azerothcore-wotlk#2397]
##### SMALL DESCRIPTION:
The fire damage radius of various braziers and other similar game objects is too large.

##### EXPECTED BLIZZLIKE BEHAVIOUR:
You should only get fire damage if standing very close to the braziers.

##### CURRENT BEHAVIOUR:
You suffer fire damage even when standing 1-2 yards away from the game object.

##### STEPS TO REPRODUCE THE PROBLEM:
- near Thrall in Orgrimmar (`.go 1923.89 -4198.55 42.1597 1`)
- in the cave in Valley of Trials (`.go -605.034 -4172.43 41.0001 1`)

##### EXTRA NOTES:
none

##### BRANCH(ES):
master

##### AC HASH/COMMIT:
2cb28e5f5342e92934b5d7d3acd771a7785be82d

##### OPERATING SYSTEM:
Ubuntu 16.04

##### MODULES:
mod-autobalance
mod-npc-beastmaster
mod-transmog
mod-weapon-visual

##### OTHER CUSTOMIZATIONS:
none

**Comment by Maurowin** [comment]
All fire, whether it's chimney or even campfires the radius is too wide

**Comment by pklloveyou** [comment]
@Stoabrogga 
![image](https://user-images.githubusercontent.com/18535853/68267045-f3dca800-008b-11ea-91eb-597036031398.png)

**Comment by pklloveyou** [comment]
I don't know if it's a hack.But it works.

**Comment by FrancescoBorzi** [comment]
@pklloveyou http://www.azerothcore.org/wiki/How-to-create-a-PR

**Comment by Stoabrogga** [comment]
Hmm, I think this issue can be closed. According to Spell.dbc the EffectRadiusIndex is 7 for spell 7897 and 15 for 7902. When looking up those indices in SpellRadius.dbc then 7 is 2 yards radius and 15 is 3 yards radius. So this seems to be blizzlike as it is now, although it looks strange to me.

**Comment by BarbzYHOOL** [comment]
if it makes no sense i'm for fixing it xD

**Comment by Stoabrogga** [comment]
I think this can be fixed much easier:
```sql
UPDATE `gameobject_template` SET `Data2` = 1 WHERE `entry` = 2061;
```
After applying this it looks good to me.

**Comment by BarbzYHOOL** [comment]
what is it??

**Comment by Stoabrogga** [comment]
Reducing the diameter when the trap game object should trigger from 2 to 1 yards.

**Comment by BarbzYHOOL** [comment]
good no?

**Comment by Maurowin** [comment]
It's working for all fire camp too?

**Comment by Stoabrogga** [comment]
Yes, that's for every game object in the world which uses 2061 as template (it's a trap type). I've began working on a much better solution which also fixes #2396 but currently have no time to finish it for the next days (perhaps weeks).

**Comment by BarbzYHOOL** [comment]
so it's a hackfix ?

**Comment by Stoabrogga** [comment]
It does not really solve this. Although the radius is decreased, it still is big enough to hurt the player while standing away from the fire. I want to solve this by a GameObjectAI script but this will take some time. But after I'm finished it will also solve the other issue.

**Comment by Stoabrogga** [comment]
Done, see PR #2432, which should also fix #2396.

**Comment by pklloveyou** [comment]
I tested it and it worked great
